go.mod

```diff
@@ -86,3 +86,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
+
+replace github.com/aquasecurity/tracee/types => github.com/NDStrahilevitz/tracee/types v0.0.0-20220412091018-d793c93b2ebc
```

Comment:
Except this replace statement other changes look good to me.

Reply:
Yes this was for testing, now that the types commit is up I will update this.

Reply:
Done